bad_connector_woocommerce/models/woo_backend/common.py

```diff
@@ -179,6 +180,45 @@ def _get_stock_field_id(self):
     woo_setting_id = fields.Many2one(comodel_name="woo.settings")
     stock_update = fields.Boolean(related="woo_setting_id.stock_update")
     recompute_qty_step = fields.Integer(string="Recompute Quantity Batch", default=1000)
+    access_token = fields.Char(string="Access Token(Live)", readonly=True)
+    test_access_token = fields.Char(string="Access Token(Staging)", readonly=True)
+    webhook_config = fields.Html(
+        string="Webhook Configurations",
+        readonly=True,
+        default="""<div>
+        <h2>Follow these steps to set up a WooCommerce webhook for
+        bad_connector_woocommerce integration:</h2>
+        <ol>
+            <li>Navigate to WooCommerce > Settings > Advanced > Webhooks.</li>
+            <li>Click "Add Webhook" and provide the following details:</li>
+        </ol>
+        <ul>
+            <li><strong>Name:</strong> Update Product</li>
+            <li><strong>Status:</strong> Active</li>
+            <li><strong>Topic:</strong> Product Update</li>
+            <li><strong>Delivery URL:</strong>
+            http://www.website.com:8080/update_product/woo_webhook/d4ea64d3-8f85-
```

Comment:
this url has to be dymanic calculated from the web based URL and token current value